docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,119 @@
+# Move an instance from one region to another
+
+This guide will show you how to move or copy your instance volume to a different region in {{extra.brand}} Cloud.
+
+## Prerequisites
+
+- Installed OpenStack [CLI client](../../../getting-started/enable-openstack-cli/)
+- Access to source and target location (RC-file)
+- Enough space on the local machine that you download your server to
+- Target location configured with Network, Security Groups etc
+
+## Recommendations
+
+Use the UUID of the instance instead of using the instance name, this will make sure you are using the correct instance.
+
+To find the correct UUID of your instance you can use the following command:
+
+```bash
+openstack server list
+```
+
+To get the correct UUID of your instance root volume you can use the following command:
+
+```bash
+openstack server show <uuid-of-instance> -c volumes_attached
+```
+
+If there is multiple volumes attached usually the first volume is the root volume.
+
+## Creating a snapshot
+
+If you are using boot from volume servers (recommended) you first need to shutdown and delete your server, you can use the following commands:
+
+```bash
+# Shutdown your VM
+openstack server stop <uuid-of-instance>
+# Delete your server
+openstack server delete <uuid-of-instance>
```

Comment:
Uh, no. If the user has enabled `delete_on_termination` for the boot volume (which they can do both via an API call and via the GUI, on server spin-up), then this will throw the volume away, along with the server. Please don't suggest to delete the server here.

Reply:
Good point, removed this and informed customer to stop the server atleast.